fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientStatusChecker.java

```diff
@@ -25,6 +25,12 @@
 import org.slf4j.LoggerFactory;
 
 public class ClientStatusChecker {
+
+    protected ClientStatusChecker() {
+        // prevents calls from subclass(if any)
+        throw new UnsupportedOperationException();
+    }
```

Comment:
This is weird / un-usual... I think the typical solution for this is to make the constructor private, and/or (?) declare the class to be `final` so that there can be no subclasses.

```suggestion
    private ClientStatusChecker() {
    }
```

Reply:
Agreed! Making the changes